### DISCUSSION TITLE
Why is this icon overlaid on the fps counter?

### DISCUSSION THREAD

**Discussion by SheepCommander** [ppy/osu#22708] [Q&A] [answered]
There's this circle icon that doesn't do anything when I click on it, it's just obstructing the fps counter

![image](https://user-images.githubusercontent.com/66646514/220290736-a579b678-0821-4beb-9ba5-876bc04dbf06.png)

(Lazer)

**Comment by peppy** [comment, marked as answer]
I have no idea what this icon is, but it's not provided by osu! (so is likely an overlay you have running on your system).

**Reply by ynk** [reply]
It's Geforce Experience from Nvidia

**Reply by SheepCommander** [reply]
ohhhh thanks!
